src/main/webapp/app/exercises/shared/included-in-overall-score-picker/included-in-overall-score-picker.component.html

```diff
@@ -3,9 +3,9 @@
         <input ngbButton type="radio" [value]="IncludedInOverallScore.INCLUDED_COMPLETELY" /> {{ 'artemisApp.exercise.yes' | artemisTranslate }}
     </label>
     <label ngbButtonLabel class="btn-primary">
-        <input ngbButton type="radio" [value]="IncludedInOverallScore.INCLUDED_AS_BONUS" /> {{ 'artemisApp.exercise.bonus' | artemisTranslate }}
+        <input ngbButton type="radio" [value]="IncludedInOverallScore.INCLUDED_AS_BONUS" />{{ 'artemisApp.exercise.bonus' | artemisTranslate }}
     </label>
-    <label ngbButtonLabel class="btn-primary">
+    <label ngbButtonLabel class="btn-primary" *ngIf="allowNotIncluded">
```

Comment:
Since the removal of this button leaves only two buttons behind, users might be confused about which button is selected. The selected button above this element group uses dark blue, however here light blue is used. This is not in scope of this PR so I am approving this PR.

Reply:
-> #4844